docs/fastdds/dds_layer/core/policy/policy.rst

```diff
@@ -1,5 +1,2259 @@
+
+LivelinessQosPolicyKind
+"""""""""""""""""""""""
+
+There are three possible values:
+
+* **Automatic**: The service takes the responsibility for renewing the leases at the required rates, as long as the
+  local process where the participant is running and the link connecting it to remote participants exists, the
+  entities within the remote participant will be considered alive.
+  This kind is suitable for applications that only need to detect whether a remote application is still running.
+* The two **Manual** modes require that the application on the publishing side asserts the liveliness periodically
+  before the lease_duration timer expires. This action can be done by calling `assert_liveliness` member function, or
+  by writing some data.
+
+  * **Manual by Participant**: If one of the entities in the publishing side asserts its' liveliness, the service
+    deduces that all other entities within the same DomainParticipant are also alive.
+  * **Manual by Topic**: This mode is more restrictive and requires that at least one instance within the DataWriter
+    is asserted to consider that the DataWriter is alive.
+
+.. _liveliness_compatibilityrule:
+
+Compatibility Rule
+""""""""""""""""""
+
+To maintain the compatibility between LivelinessQosPolicy in DataReaders and DataWriters, the DataWriter kind must be
+higher or equal to the DataReader kind.
+And the order between the different kinds is::
+
+ AUTOMATIC < MANUAL BY PARTICIPANT < MANUAL BY TOPIC
+
+Table with the possible combinations:
+
++-----------------------+-----------------------+-----------------+
+| DataWriter kind       | DataReader kind       | Compatibility   |
++=======================+=======================+=================+
+| Automatic             | Automatic             | Yes             |
++-----------------------+-----------------------+-----------------+
+| Automatic             | Manual By Participant | No              |
++-----------------------+-----------------------+-----------------+
+| Automatic             | Manual By Topic       | No              |
++-----------------------+-----------------------+-----------------+
+| Manual By Participant | Automatic             | Yes             |
++-----------------------+-----------------------+-----------------+
+| Manual By Participant | Manual By Participant | Yes             |
++-----------------------+-----------------------+-----------------+
+| Manual By Participant | Manual By Topic       | No              |
++-----------------------+-----------------------+-----------------+
+| Manual By Topic       | Automatic             | Yes             |
++-----------------------+-----------------------+-----------------+
+| Manual By Topic       | Manual By Participant | Yes             |
++-----------------------+-----------------------+-----------------+
+| Manual By Topic       | Manual By Topic       | Yes             |
++-----------------------+-----------------------+-----------------+
```

Comment:
write values in uppercase and underscores
In addition, the names should be:
AUTOMATIC_LIVELINESS_QOS
MANUAL_BY_PARTICIPANT_LIVELINESS_QOS
MANUAL_BY_TOPIC_LIVELINESS_QOS